dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -109,7 +115,24 @@ def _add_helper_merge_profile_histograms(self, other1, other2):
         if not bin_methods:
             raise ValueError('Profiles have no overlapping bin methods and '
                              'therefore cannot be added together.')
+        elif other1.user_set_histogram_bin and other2.user_set_histogram_bin:
+            if other1.user_set_histogram_bin != other2.user_set_histogram_bin:
+                warnings.warn('User set histogram bin counts did not match. '
+                              'Choosing the larger bin count.')
+            self.user_set_histogram_bin = max(other1.user_set_histogram_bin,
+                                              other2.user_set_histogram_bin)
+
         self.histogram_bin_method_names = bin_methods
+        self.histogram_methods = dict()
+        for method in self.histogram_bin_method_names:
+            self.histogram_methods[method] = {
+                'total_loss': 0,
+                'current_loss': 0,
+                'histogram': {
+                    'bin_counts': None,
+                    'bin_edges': None
+                }
+            }
```

Comment:
This could use a comment explaining why this is repeated for the case of 'custom', and also maybe an if statement to ensure it is being done in that case, and signal the intent for that case.

Reply:
Actually this still makes sense in the non-'custom' case, but still a comment would be helpful to clarify the need for seemingly-redundant computation.